src/hpr_max_copies.erl

```diff
@@ -0,0 +1,140 @@
+-module(hpr_max_copies).
+
+-export([
+    init/0,
+    check/2,
+    delete/1
+]).
+
+-define(ETS, hpr_max_copies_ets).
+-define(MAX_COPIES, max_copies).
+-define(MAX_TOO_LOW, max_copies_max_too_low).
+-define(CLEANUP, timer:hours(1)).
+-define(CLEANUP_SLEEP, timer:hours(1)).
+
+-spec init() -> ok.
+init() ->
+    ets:new(?ETS, [
```

Comment:
Document the format of a row?
How do values get in the table?

Reply:
Oh, I see how values get in. I hadn't realized `update_counter` adds missing rows.